cmd/app/app.go

```diff
@@ -85,6 +85,12 @@ func NewCommand(ctx context.Context) *cobra.Command {
 				return fmt.Errorf("failed to setup driver: " + err.Error())
 			}
 
+			go func() {
+				<-ctx.Done()
+				log.Info("shutting down driver", "context", ctx.Err())
+				d.Stop()
```

Comment:
Is there time for this to be meaningfully shut down before the app exits?

Reply:
Probably not. This should be fixed.

Reply:
I guess the standard K8s handling of quitting gracefully on first SIGTERM and hard quitting on the second would work

Reply:
I think it is fine actually
https://github.com/kubernetes-sigs/controller-runtime/blob/master/pkg/manager/signals/signal.go